src/orga/subteams.rst

```diff
@@ -2,6 +2,66 @@ A list of current sub-teams
 ***************************
 
 
+Code of Conduct Sub-Team
+=============================
+
+Role
+----
+The code-of-conduct (CoC) sub-team is responsible for accepting, handling
+and resolving code-of-conduct related items (including but not limited to
+CoC violations, enforcement, communications, warnings, etc.) on conda-forge's
+communication channels (including but not limited to Gitter, GitHub, Keybase, the various
+mailing lists, etc).
+
+Charter
+-------
+Static
+
+Responsibility
+--------------
+The CoC sub-team will operate via the following rules:
+
+- It can only be comprised of ``core`` members.
+- Any ``core`` member is welcome to participate.
+- ``core`` members involved in CoC reports will use their best judgment to recuse
+  themselves in cases where there may be or even appear to be a conflict of interest.
+- The CoC team will judge CoC-related issues according to the current official
+  conda-forge CoC. The ``conda-forge`` CoC is subject to change by the wider ``core`` group.
+- The team may seek guidance from NumFocus and the wider ``core`` group as needed.
+- Due to the potentially sensitive nature of CoC violations, the CoC team will
+  strive to preserve the confidentiality of involved persons as much as possible.
```

Comment:
We may need to be more prescriptive here as well. For instance, how much information is held back from core more generally? In which situations could information be released and to whom?